server/src/controllers/projects/tasks/getTasks.ts

```diff
@@ -0,0 +1,15 @@
+import { Response } from 'express';
+import getTasksQuery from '../../../database/ quieres';
+
+const getTasks = async (req: any, res: Response) => {
+  const { userId } = req.userInformation;
```

Comment:
Here you can replace "any" type with "Request" and import it from "express",
as you did in (res: Response)